featuretools/entityset/deserialize.py

```diff
@@ -36,7 +36,8 @@ def description_to_variable(description, entity=None):
         kwargs = {} if is_type_string else description['type']
         variable = variable(description['id'], entity, **kwargs)
         variable.interesting_values = description['properties']['interesting_values']
-    return variable
+        return variable
+    return (variable, description['properties']['interesting_values'])
```

Comment:
Makes sense. Before, interesting values were being lost and it didn't seem to make sense to loop through all the variables again when we didn't need to because we did it here.